app/res/values/arrays.xml

```diff
@@ -75,5 +75,20 @@
 		<item>Hamlie</item>
 		<item>Nehasie</item>
 		<item>Pagumien</item>
-	</string-array>    
+	</string-array>   
+	<string-array
+	    name="nepali_months">
```

Comment:
Is there a localized version of this in Devanagari script for apps that are using the phone in native script?